### DISCUSSION TITLE
Missing field in form submission

### DISCUSSION THREAD

**Discussion by PixelTaro** [OpnForm/OpnForm#440] [Q&A]
I have custom code dropdown field name "Document Type" and it takes value from an API. Everything works well but when I submitted the form, I cannot find the selected "Document Type" or its column anywhere on Form Submission. I'm not sure if I miss something. Please help.

![WhatsApp Image 2024-06-06 at 14 17 53](https://github.com/JhumanJ/OpnForm/assets/102939411/0b1a5856-ba40-43d3-9a90-61ac070e5811)
![WhatsApp Image 2024-06-06 at 14 18 20](https://github.com/JhumanJ/OpnForm/assets/102939411/503bf2c4-1c91-45c5-a3a8-7d908724f867)
![WhatsApp Image 2024-06-06 at 14 18 20 (1)](https://github.com/JhumanJ/OpnForm/assets/102939411/90cbf8a7-545f-4bf8-83c2-15218971942b)

**Comment by JhumanJ** [comment]
Hi, is this self hosted? Can you share a form URL maybe? I guess your custom code somehow broke the field. Anything in the console maybe?

**Reply by PixelTaro** [reply]
> Hi, is this self hosted? Can you share a form URL maybe? I guess your custom code somehow broke the field. Anything in the console maybe?

No, it's hosted on opnform.com. Here is the form link: https://opnform.com/forms/toilet-feedback-form-a3pa5k

**Comment by JhumanJ** [comment]
Sorry but we can't provide help for custom code - it seems like there's an issue with custom code here unfortunately.
